src/App.tsx

```diff
@@ -31,16 +31,16 @@ function App() {
   const [getFilmQuery] = StarWarsAPI.endpoints.getFilm.useLazyQuery()
   const [getPlanetQuery] = StarWarsAPI.endpoints.getPlanet.useLazyQuery()
 
-  const handleOpen = () => setPeopleDialogOpen(true)
-  const handleClose = () => setPeopleDialogOpen(false)
+  const handleOpen = () => setPeopleDialogOpen(false)
+  const handleClose = () => setPeopleDialogOpen(true)
```

Comment:
**Suggestion:** Correct the logic of handleOpen and handleClose functions. Currently, they are doing the opposite of what their names suggest.
```suggestion
  const handleOpen = () => setPeopleDialogOpen(true)
  const handleClose = () => setPeopleDialogOpen(false)
```